action.yml

```
@@ -23,7 +23,7 @@ inputs:
     default: https://api.tower.nf
   pipeline:
     description: Pipeline repository
-    default: ${{ github.repository }}
+    default: https://github.com/${{ github.repository }}
```

isn't there a github variable to get the full URL?

---

yeah good question, that would be nicer 👍🏻

---

only this I could find was the GITHUB_SERVER_URL in the https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables but that only gives the `https://github.com` part

---

There's `github.repositoryUrl` which gives the _git_ address, eg. `git://github.com/codertocat/hello-world.git` - I'm not sure that this will work though ([docs](https://docs.github.com/en/actions/learn-github-actions/contexts)).

Ah there is also `github.server_url` which gives `https://github.com`. So can do this:

```suggestion
    default: ${{github.server_url}}/${{ github.repository }}
```

Not sure if it's actually all that much nicer 😅 But will mean that the action should work on enterprise versions of GitHub I guess..

---

I think we found the same thing at the same time!